### ISSUE TITLE
How about a biff/xbiff feature?

### ISSUE THREAD

**Issue by joe-average-user** [RainLoop/rainloop-webmail#1508]
This is of course more of an enhancement request. It would be very handy to be notified of new mails in the configured accounts somehow, be it in the menu on top right, or even a new account list in the bottom left. Else you constantly have to change accounts to see if a new mail has arrived somewhere...

**Comment by megadr01d** [comment]
Dupe of https://github.com/RainLoop/rainloop-webmail/issues/1359.

**Comment by Lewiscowles1986** [comment]
should this & #1359 which is a dupe of #377 not both be closed if duplicate?

**Comment by megadr01d** [comment]
Yes they should, but unfortunately GitHub is full of users who don't care to track and be responsible for closing their own issues. Too many invalid, fixed or duplicate open issues just give the project (Rainloop, in this case) a bad reputation and a sense of abandonment, when it's not.

**Comment by joe-average-user** [comment]
Really, the sense of abandonment probably comes from an issue/feature that is requested since 2014 (three years ago). Closing the latest issue will only let it vanish for another year(s) from sight.
Strangely I wouldn't say rainloop is no good in terms of maintenance. Actually all my other issues have been dealt with very shortly. This though is a bit strange as there is no single argument against it up to now...

**Comment by megadr01d** [comment]
> Really, the sense of abandonment probably comes from an issue/feature that is requested since 2014 (three years ago).

A feature request is what it is, a request which can be accepted or not. If I make a feature request and years later it's still not addressed, I close the issue myself.

> Closing the latest issue will only let it vanish for another year(s) from sight.

It's a duplicate. In this case, a duplicate of a duplicate. It's just unwanted noise.
So why it's not out of sight from me? Am I special? Do I have super powers? No, I use GH's issue search.
